gwctl/.goreleaser.yaml

```diff
@@ -0,0 +1,35 @@
+# https://goreleaser.com/
+#
+# For local testing, use:
+# 
+#   goreleaser release --snapshot --clean
+project_name: gwctl
```

Comment:
Do we intend to keep gwctl as a standalone binary or as a kubectl plugin? If it is both, then wouldn't it be better to name the binary `kubectl-*`?

Reply:
This PR only intended to make this available as a standalone binary, but you are right for kubectl plugin that is indeed a better name. (I'll attempt at addressing this in the new PR)